RedHat-UX/red-hat-design-system
# chore: update to @patternfly/elements rc

## What I did

1. install `@patternfly/elements`
2. replace instances of pfe imports and elements
3. upgrade to pfe-tools 1.0 rc
3. upgrade to 11ty 2.0 beta
4. refactor 11ty config
5. nice up the `importMap.cjs` generator for 11ty :crossed_fingers: hopefully this is the one.
6. migrate from npm-run-all to wireit
3. upgrade dependencies


## Testing Instructions

1. check changed docs and dev server demos
2. `git clean -dfx && npm ci && npm start` should run both the wds dev server and the 11ty dev server successfully. warning: this git clean is a destructive operation

Check all these DP demos:

1. [Accordion main demo](https://deploy-preview-725--red-hat-design-system.netlify.app/components/accordion/demo/)
1. [Accordion context nested demo](https://deploy-preview-725--red-hat-design-system.netlify.app/components/accordion/demo/color-context/)
1. [Context Provider main demo](https://deploy-preview-725--red-hat-design-system.netlify.app/components/context-provider/demo/)
1. [Context Provider context nested demo](https://deploy-preview-725--red-hat-design-system.netlify.app/components/context-provider/demo/nested-contexts/)
1. [Footer main demo](https://deploy-preview-725--red-hat-design-system.netlify.app/components/footer/demo/) (social icons)
1. [Global footer nested demo](https://deploy-preview-725--red-hat-design-system.netlify.app/components/footer/demo/global-footer/) (light dom css)
1. [playgrounds](https://deploy-preview-725--red-hat-design-system.netlify.app/components/footer/#demo) (switching playground tabs)
1. [Call to Action demo](https://deploy-preview-725--red-hat-design-system.netlify.app/components/call-to-action/demo/)
1. [Dialog demo](https://deploy-preview-725--red-hat-design-system.netlify.app/components/dialog/demo/)
1. [Stat demo](https://deploy-preview-725--red-hat-design-system.netlify.app/components/statistic/demo/)
1. [Tooltip demo](https://deploy-preview-725--red-hat-design-system.netlify.app/components/tooltip/demo/)

## Notes to Reviewers
- [x] Blocked by https://github.com/patternfly/patternfly-elements/pull/2342
- [x] Blocked by https://github.com/patternfly/patternfly-elements/pull/2358
- [x] Blocked by https://github.com/patternfly/patternfly-elements/pull/2360
- [x] Blocked by https://github.com/patternfly/patternfly-elements/pull/2362
- [x] Blocked by https://github.com/patternfly/patternfly-elements/pull/2367
- even if CI is green, check the test log to make sure all tests indeed ran and passed
